spec/controllers/api/v1/beneficiaries_controller_spec.rb

```diff
@@ -0,0 +1,150 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::BeneficiariesController, type: :controller do
+  let!(:beneficiary) { create :beneficiary }
```

Comment:
What happens if you change this from `let!(:beneficiary)` to `let(:beneficiary)`? Might save object creation in a few tests, then again, it might cause a test that depends on it's pre-existence, to fail. Worth a check.

Reply:
No apparent changes when switching it